sdk/core/core-websockets/package.json

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "@azure/core-websockets",
+  "version": "1.0.0-beta.1",
+  "description": "Implementation of a websockets client for Node.js and browsers.",
```

Comment:
Nit: I am not too sure about this. Should we call this implementation of websockets client, or enhancement?

Reply:
I am not sure if I follow but I am open to suggestions.